src/components/ConferenceList.js

```diff
@@ -2,33 +2,74 @@ import { connect } from 'react-redux';
 import React from 'react';
 import ConferenceCard from './ConferenceCard';
 import SearchBar from './SearchBar';
+import DropDown from './DropDown';
 import { fetchConferenceData } from '../redux/actions';
 
 export class ConferenceList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedValue: 'default'
+    };
+    this.handleDropDown = this.handleDropDown.bind(this);
+  }
+
   componentDidMount() {
     this.props.fetchData();
   }
 
+  searchMatches = field =>
+    field.toLowerCase().search(this.props.searchParam.toLowerCase()) !== -1;
+
+  handleDropDown = event => {
+    this.setState({ selectedValue: event.target.value });
+  };
+
   render() {
     const filteredList = this.props.conferences.filter(
       item =>
-        item.eventName
-          .toLowerCase()
-          .search(this.props.searchParam.toLowerCase()) !== -1
+        this.searchMatches(item.eventName) ||
+        this.searchMatches(item.eventLocation)
     );
-    console.log(this.props.conferences);
-    return (
-      <section className=" mw5 mw7-ns center">
-        <h2 className=" w-100 sans-serif pa0 f2 tl fw2 mh0 mt4 mb3">
-          Upcoming Conferences
-        </h2>
-        <SearchBar />
-        {filteredList &&
-          filteredList.map(conference => (
+
+    const filteredByDate = this.props.conferences.filter(
+      conference => new Date(conference.submissionDueDate) - new Date() < 7
+    );
+
+    if (this.state.selectedValue === 'default') {
+      return (
+        <section className=" mw5 mw7-ns center">
```

Comment:
Since we are only changing the title and the list when the dropdown value changes, I think it would make sense to add the conditional around these two things. The other things, like the section, SearchBar, and Dropdown, are the same so we don't need to conditionally render them.